packages/plugin-core/src/constants.ts

```diff
@@ -16,6 +16,7 @@ export enum DendronContext {
   HAS_CUSTOM_MARKDOWN_VIEW = "hasCustomMarkdownPreview",
   NOTE_LOOK_UP_ACTIVE = "dendron:noteLookupActive",
   BACKLINKS_SORT_ORDER = "dendron:backlinksSortOrder",
+  ENABLEEXPORTPODV2 = "dendron:enableExportPodV2",
```

Comment:
super nitpick but this should be UPPER_SNAKE_CASE

Reply:
not nitpick. we should be consistent about this